repaygithub/cactus
# CACTUS-549 :: Mobile Breadcrumbs

Ticket: https://repayonline.atlassian.net/browse/CACTUS-549

This was definitely a challenge, mostly because devs can use custom components for their links. You'll probably notice some workarounds that I implemented to get this to work, but if you have any ideas for how I could do this better, don't hesitate to suggest them; I know there are probably ways to implement this better that I didn't think of.

### Testing
1. Open the "Custom Item Elements" Breadcrumb story and set the viewport to a tiny screen size.
2. Make sure you can select a link using your mouse, both by clicking on the first breadcrumb and the ellipsis in the middle.
3. Make sure you can select a link using the keyboard, triggering the dropdown from both the first breadcrumb and the ellipsis.
4. Repeat steps 1-3 using the "Add more Breadcrumbs" story.
5. Set your viewport back to normal and make sure the breadcrumbs still work as they did before on larger screen sizes.
6. Do a quick check on the shape and box shadow of the dropdowns in the ColorPicker, DateInput, and MenuButton to make sure the refactoring didn't break anything there.
